connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorValidationIntegrationTest.java

```diff
@@ -549,12 +549,7 @@ public ConfigDef config() {
         }
     }
 
-    public static class TestConverterWithNoConfigDef extends TestConverter {
-        @Override
-        public ConfigDef config() {
-            return null;
-        }
-    }
+    public static class TestConverterWithNoConfigDef extends TestConverter { }
```

Comment:
While it copes the behavior the the super class, I'd prefer keeping the current code to keep things easy to understand.

Reply:
Ok, reversed the changes.